Tests/NIOOpenSSLTests/OpenSSLIntegrationTest.swift

```diff
@@ -288,11 +295,35 @@ class OpenSSLIntegrationTest: XCTestCase {
         return try assertNoThrowWithValue(SSLContext(configuration: config), file: file, line: line)
     }
 
+    private func configuredSSLContext<T: Collection>(passphraseCallback: @escaping OpenSSLPassphraseCallback<T>,
+                                                     file: StaticString = #file, line: UInt = #line) throws -> NIOOpenSSL.SSLContext
+                                                     where T.Element == UInt8 {
+        let config = TLSConfiguration.forServer(certificateChain: [.certificate(OpenSSLIntegrationTest.cert)],
+                                                privateKey: .file(OpenSSLIntegrationTest.encryptedKeyPath),
+                                                trustRoots: .certificates([OpenSSLIntegrationTest.cert]))
+        return try assertNoThrowWithValue(SSLContext(configuration: config, passphraseCallback: passphraseCallback), file: file, line: line)
```

Comment:
why doesn't the `assertNoThrowWithValue` not do `file: StaticString = #file, line: Int = #line` to have file and line taken automatically using default args?

Reply:
It does, but here we are propagating the file and line from higher up so they have to be passed explicitly.